updates to show_repo

yonicd opened the pull request.

Commit:
add OS switch for show_repo in git archive call

Commit:
keep _archive_.txt in folder and add param to update it, important for big repos

Commit:
use git ls-tree instead of git archive in show_repo

Commit:
run show_repo from relative path and return silent outputs from show_repo and search_ghap

Commit:
update documentation in show_repo

yonicd commented:
the conflicts were created because i added an invisible return to `search_ghap` and `show_repo`

Commit:
search builder for ghap meta data

Commit:
cferateDB on app start

Commit:
add fork info for shiny app

hafen commented:
Thanks. Sorry I'm slow - working on some time-sensitive stuff. I hope to review this tomorrow.

Commit:
update sparse_ghap

yonicd committed:
Create global.R

yonicd committed:
Create server.R

yonicd committed:
Create global.R

yonicd committed:
Create global.R

yonicd committed:
Create global.R

yonicd committed:
Create global.R

yonicd committed:
Create global.R

Commit:
create www for filters.Rdata

yonicd committed:
Create server.R

Commit:
add meta_ghap to searchbuilder

Commit:
add jstree app to inst/examples

Commit:
set shinyapp path to get_git_basepath

Commit:
update sparse_ghap to work with existing non-sparse repos

Commit:
update searchbuilder app to include the jstree functionality

Commit:
switch querybuilder to yonicd /querybuilder repository

Commit:
remove browser() from script.R

Commit:
add vcs and jsTree to installed lib checks for the searchbuilder app

Commit:
remove help function idtoname

Commit:
add selectize filtering to querybuilder

Commit:
add mrg dependencies to description file

yonicd committed:
update ghap package

yonicd committed:
remove mrg remotes

yonicd committed:
return mrg remotes

Commit:
check to see that all remote pkgs are up to date in searchbuilder init

Commit:
Merge branch 'master' of https://github.com/yonicd/ghap

Commit:
create addin for searchbuilder in ghap package

Commit:
bump version

Commit:
add check for `.git/info/sparse-checkout` when moving from nonsparse to sparse repo in use_study()

yonicd committed:
Update DESCRIPTION

yonicd committed:
Update DESCRIPTION

yonicd committed:
update searchbuilder

yonicd committed:
fix spacing in searchbuilder

yonicd committed:
fix package dependencies

Commit:
add DT namespace to dataTableAjax

Commit:
update searchbuilder

Commit:
remove searchbuilder from ghap package. add example of use_study with sparse definitions

Commit:
merge differences from upstream

Merge remote-tracking branch 'upstream/master'

# Conflicts:
#	DESCRIPTION
#	NAMESPACE
#	R/asdtm_utilities.R
#	R/git.R
#	R/met_ghap-data.R
#	R/search_ghap.R
#	R/show_repo.R
#	R/sparse_ghap.R
#	man/meta_ghap.Rd
#	man/use_study.Rd

Commit:
remove miscellaneous files and clean up description file
